cmd/daemon/bootstrap.go

```diff
@@ -0,0 +1,81 @@
+//go:build windows
+
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/openshift/windows-machine-config-operator/pkg/daemon/controller"
+	"github.com/openshift/windows-machine-config-operator/pkg/daemon/winsvc"
+)
+
+var (
+	bootstrapCmd = &cobra.Command{
+		Use:   "bootstrap",
+		Short: "Starts required Windows services to bootstrap a Node.",
+		Long: "Starts all Windows services on an instance that are pre-requisites for Node object creation, " +
+			"configured according to the information given by the Windows services ConfigMap.",
+		Run: runBootstrapCmd,
+	}
+	desiredVersion string
+)
+
+func init() {
+	rootCmd.AddCommand(bootstrapCmd)
+	bootstrapCmd.PersistentFlags().StringVar(&desiredVersion, "desired-version", "",
+		"Version of the services ConfigMap to use as the source of truth for service configuration")
+	bootstrapCmd.MarkPersistentFlagRequired("desired-version")
+}
+
+// runBootstrapCmd starts all Windows services that are pre-requisites for node object creation taken from encoded arg
```

Comment:
Sorry, encoded arg doesn't exist anymore. I'll fix this comment after a round of reviews